server/apps/distributed-app/sample-configuration/opensearch.properties

```diff
@@ -94,3 +94,7 @@ opensearch.indexAttachments=true
 # Please note that custom overrides can be defined here.
 #
 # opensearch.search.overrides=org.apache.james.mailbox.cassandra.search.AllSearchOverride,org.apache.james.mailbox.cassandra.search.DeletedSearchOverride, org.apache.james.mailbox.cassandra.search.DeletedWithRangeSearchOverride,org.apache.james.mailbox.cassandra.search.NotDeletedWithRangeSearchOverride,org.apache.james.mailbox.cassandra.search.UidSearchOverride,org.apache.james.mailbox.cassandra.search.UnseenSearchOverride
+
+# Optional. Default is `false`
+# When set to true, the message will be tried to re-index from the indexed message when moved. If the message is not found, it will fall back to old behavior (The message will be indexed from the blobStore source)
```

Comment:
```suggestion
# When set to true, James will attempt to reindex from the indexed message when moved. If the message is not found, it will fall back to the old behavior (The message will be indexed from the blobStore source)
```